src/bio/bio_xstream.c

```diff
@@ -1554,6 +1557,88 @@ bio_xsctxt_free(struct bio_xs_context *ctxt)
 	D_FREE(ctxt);
 }
 
+struct subsystem_init_arg {
+	struct common_cp_arg *cp_arg;
+	void                 *json_data;
+	size_t                json_data_size;
+};
+
+static void
+subsystem_init_arg_fini(void *arg, int rc)
```

Comment:
hmm, why do we need this callback? Can't we just modify existing subsys_init_cb()?

Reply:
yeah, indeed, we can use that cb. Thanks.